tests/input/cpp/bug_858.cpp

```diff
@@ -0,0 +1,10 @@
+enum {
```

Comment:
For the sake of completeness, maybe this test should also have (line breaks elided for brevity):

```c++
enum { x, y };
enum { x, y=0 };
enum { x, y=0 /*comment*/ };
```

...both with and without trailing comma; IOW, test all combinations of the final enum value a) followed or not by a comment, b) having an explicit value or not, c) followed or not by a comma?